src/consensus/activation.cpp

```diff
@@ -27,3 +27,18 @@ bool IsLeviticusEnabled(const Consensus::Params &params,
     return pindexPrev->GetMedianTimePast() >=
            gArgs.GetArg("-leviticusactivationtime", params.leviticusActivationTime);
 }
+
+bool IsNumbersEnabled(const Consensus::Params &params,
```

Comment:
adding this as some locations only have access to the height, and not the pindex.

Reply:
where is this needed?

If you're 100% sure it's needed, also add some tests

Reply:
There is a test. You can look at ABC code for now it's needed. But GetNextBlockScriptFlags() is where it's used.